web/app/test/ServiceMeshTest.jsx

```diff
@@ -38,32 +38,34 @@ describe('ServiceMesh', () => {
     });
   });
 
-  it("displays message for no deployments detetcted", () => {
+  it("renders the spinner before metrics are loaded", () => {
     fetchStub.returnsPromise().resolves({
       ok: true,
-      json: () => Promise.resolve({ pods: []})
+      json: () => Promise.resolve({ metrics: [] })
     });
```

Comment:
I don't think this stub is ever being invoked, since you're not calling `withPromise` as part of this test. So I'd recommend omitting it to avoid confusion.